client/src/pages/learn/full-stack-developer/review-python/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to the Python Review
+block: review-python
+superBlock: full-stack-developer
+---
+
+## Introduction to the Python Review
+
+Review Python concepts to prepare for the upcoming quiz.
```

Comment:
same here. this looks like a review page for the chapter exam instead of a quiz.

```suggestion
Review Python concepts to prepare for the upcoming exam.
```